### ISSUE TITLE
Plugin Error on Startup

### ISSUE THREAD

**Issue by bluebonanzapilot** [jhurliman/homebridge-motionblinds#7]
Getting this Error on Startup

[7/4/2021, 4:07:57 PM] [MOTION Blinds] Launched child bridge with PID 11704
[7/4/2021, 4:07:57 PM] Registering platform 'homebridge-motionblinds.MotionBlinds'
[7/4/2021, 4:07:57 PM] [MOTION Blinds] Loaded homebridge-motionblinds v1.2.0 child bridge successfully
Preparing Advertiser for 'MOTION Blinds EEAC' using bonjour-hap backend!
Starting to advertise 'MOTION Blinds EEAC' using bonjour-hap backend!
[7/4/2021, 4:07:57 PM] Homebridge v1.3.4 (MOTION Blinds) is running on port 31469.
Error: bind EADDRNOTAVAIL 238.0.0.18:32101
    at dgram.js:337:20
    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[7/4/2021, 4:07:57 PM] [MOTION Blinds] Child bridge process ended
[7/4/2021, 4:07:57 PM] [MOTION Blinds] Process Ended. Code: 1, Signal: null

**Comment by jhurliman** [comment]
It looks like something else is listening for multicast DNS packets on the same computer. What other homebridge plugins are you running? Or is there anything else running on the computer that you know is listening to multicast DNS?

**Comment by bluebonanzapilot** [comment]
I have several plugins running. I thought it might have been the Homebridge Wiz Lan Plugin, but disabling the plugin did not resolve the problem. I went though each plugin and disabled one at a time to see if the issue could be resolved however, nothing resolved the issue. The only other item running on the computer that could possibly be listening to multicast (that I know of) would be Plex Media Server which I also disabled to see if it would resolve the issue but there was no change. I did restart homebridge in between disabling each plugin.
